### DISCUSSION TITLE
[Bug]: Router Error while updating from 6.3 to 6.8

### DISCUSSION THREAD

**Discussion by sydtv** [pimcore/pimcore#9733] [Q&A / Support] [answered]
### Expected behavior

When you call domain.tld/admin, you can login to your backend. In the backend you can edit your stuff.

### Actual behavior

The login form appears and we are able to send the form. After the successful login, we get redirected to the backend, where we see de three loading dots in an infinite animation.

There are multiple console errors:

> GET http://domain.tld/js/routing?callback=fos.Router.setData net::ERR_ABORTED 500 (Internal Server Error)
> router.js:272 Uncaught Error: The route "pimcore_admin_document_document_doctypesget" does not exist.
> router.js:272 Uncaught Error: The route "pimcore_admin_misc_ping" does not exist.
> router.js:272 Uncaught Error: The route "pimcore_admin_notification_findlastunread" does not exist.

### Steps to reproduce

We did an update from 6.3 to 6.8, the problem appears since 6.6

**Comment by brusch** [comment]
Migrated from issue to discussion -> Pimcore 6 EOL. It also looks like an individual problem.

**Comment by dvesh3** [comment]
@sydtv fosjsrouting was introduced in 6.6.x version and there were fixes in following versions. you can check here https://github.com/pimcore/pimcore/pulls?q=is%3Apr+fosjsrouting+is%3Aclosed+

**Comment by sydtv** [comment, marked as answer]
It was an internal problem.
